common/dtd/ldmlSupplemental.dtd

```diff
@@ -1154,3 +1154,28 @@ Except as contained in this notice, the name of a copyright holder shall not be
 <!ELEMENT languageGroup ( #PCDATA ) >
 <!ATTLIST languageGroup parent NMTOKEN #REQUIRED >
 	<!--@MATCH:validity/language-->
+
+<!-- # Grammatical Features -->
+<!ELEMENT grammaticalData ( grammaticalFeatures* )>
+
+<!ELEMENT grammaticalFeatures ( grammaticalCase?, grammaticalGender?, grammaticalDefiniteness?, grammaticalState? ) >
+<!ATTLIST grammaticalFeatures targets NMTOKENS #REQUIRED >
+    <!--@MATCH:set/literal/nominal-->
+<!ATTLIST grammaticalFeatures locales NMTOKENS #REQUIRED >
+    <!--@MATCH:set/validity/language-->
+
+<!ELEMENT grammaticalCase EMPTY>
+<!ATTLIST grammaticalCase values NMTOKENS #REQUIRED >
+    <!--@MATCH:set/literal/ablative, accusative, comitative, dative, ergative, genitive, instrumental, locative, locativecopulative, nominative, oblique, partitive, prepositional, sociative, vocative-->
```

Comment:
We've said that these values would defined in http://linguistics-ontology.org/gold/2010

If any are not, we can allow them, but we need to have an explicit list in the ticket with clear definitions so that we can explicitly document those exceptions.

I can't find:

Case
locativecopulative
repositional
sociative

Gender
common
personal

Definiteness
_all_

State
_all_

Caveats: It could be that they are known by different terms. Also, this is just checking with a simple spreadsheet using the current values; will share with you.